TACC/tup-ui
# enhance: tup-463 login should use core-styles

## Overview

Delete CSS module styles that mimic Core-Styles.

<details>

- Some Core-Styles CSS was added after TUP login page was developed.
- Now that it is available, TUP login page can use it. Less module CSS!

</details>

## Related

- [TUP-463](https://jira.tacc.utexas.edu/browse/TUP-463)

## Changes

- **removes** `Login.module.css` styles that are redundant
- **changes** `<Login>` to rely mostly on Core-Styles

## Testing

1. Open login form.
2. Verify page styles* match production.
3. Force error to appear.
4. Verify page styles* match production.
5. Resize window.
6. Verify page styles* match production.

\* The page styles are the box and stuff around the form. Form styles were changed and tested via #376.

## UI

### Looks Like Prod

| before | after | prod |
| - | - | - |
| ![login page BEFORE](https://github.com/TACC/tup-ui/assets/62723358/8ca927f7-d2c3-4cf6-a8de-f3963fa1bd05) | ![login page AFTER](https://github.com/TACC/tup-ui/assets/62723358/3dd42b2f-1e45-4147-b452-191fc8af485d) | ![login page PROD](https://github.com/TACC/tup-ui/assets/62723358/f6bc8392-520e-4bae-87e6-733f9ec39021) |

### Use Core-Styles

| before (css module) | after (core-styles) |
| - | - |
| ![login page BEFORE uses css module](https://github.com/TACC/tup-ui/assets/62723358/cdbb945c-8bbd-4b9f-b939-5a6a62c5ed23) | ![login page AFTER uses core-styles](https://github.com/TACC/tup-ui/assets/62723358/83935fa8-d862-4158-a7f0-44d77fa3d0eb) |
